README.md

```diff
@@ -34,10 +33,7 @@ To retrieve the freshly generated DID:
 
 ### Requirements
 
-* Linux
-  * g++ to build native binaries
-* Mac OS X
-  * X Code 9 to build native binaries
+* Linux or Mac OS X
```

Comment:
The previous Linux/MacOS list was there to mention the build tools.  If there are no native deps then code should work on any Node.js platform and no need to list OSes in particular.
```suggestion
```

Reply:
I hear you. I'll change the wording to 'Only tested on Linux and Windows'. :)

Reply:
Just say nothing.  It not working on any Node.js platform is a bug somewhere.